.github/workflows/biothings_annotator_tests.yml

```diff
@@ -0,0 +1,43 @@
+name: Biothings Annotator Tests
+
+on:
+  push: {}
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run_user_geneset_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ "3.9", "3.10", "3.11", "3.12" ]
+    steps:
+      - name: Checkout biothings_annotator source
+        uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Upgrade pip
+        run: pip install --upgrade pip
+      - name: Create Python Virtual Environment
+        run: python -m venv .venv
+      - name: Source Python Virtual Environment
+        run: source .venv/bin/activate
+      - name: Install PyTest
+        run: pip install pytest
+      - name: Install Sanic
+        run: pip install sanic
+      - name: Install biothings_client
+        run: pip install biothings_client
+      - name: Install brotli
+        run: pip install brotli
+      - name: Install sanic_testing
+        run: pip install sanic_testing
+      - name: Run Tests - Curie
+        run: python -m pytest tests/test_curie.py
+      # - name: Setup tmate debug session on failure
```

Comment:
What does this commented action do?

Reply:
This is to enable the tmate debug session in case of a needed debug. When tmate is enabled we can connect to the github action container using ssh.

I usually leave it there to be easy to uncomment and enable tmate. However, we can remove it!

Reply:
Gotcha I'm fine leaving it commented. Maybe we just include a comment referencing the documentation about it or a one-liner explaining why it's commented like you just stated

Reply:
There are few tests failing, after changing to run all tests from the `tests/` directory.